### ISSUE TITLE
Add a non-interactive version for --fix

### ISSUE THREAD

**Issue by thomasfr** [QuiiBz/sherif#68]
In order to run sherif in pre-commit hooks for example, it would be necessary to be able to run `sherif --fix` in non-interactive mode. For instance by specifying which version it should change it to (e.g --highest, --lowest, --in-between) together with --fix it should run in non-interactive mode and not ask for any inputs, just fix it.

**Comment by QuiiBz** [comment]
Auto-fixing the dependencies would require to refresh the lockfile, which the user might forgot to do if Sherif was run automatically via a pre-commit hook (unless we do https://github.com/QuiiBz/sherif/issues/38#issuecomment-2198020817).

**Comment by thomasfr** [comment]
That’s true, you are right. Haven’t thought about that. Well I guess then --fix should implicitly also call --install automatically.

**Comment by jayalfredprufrock** [comment]
Would love to see this included! I've had issues running sherif with --fix flag in combination with lefthook whenever it hits a package version mismatch and requires interaction. For some reason, the way sherif utilizes stdin doesn't seem to be compatible with lefthook, even with its `interactive` option.

**Comment by QuiiBz** [comment]
Since we implemented automatic install when running `--fix`, we can now add a new flag to auto select the highest or lowest dependency version in the workspace. We cannot easily implement an in-between version as there might be multiple in-between versions, so Sherif wouldn't know which one to choose.
